packages/plugins/rubygem-redhat_access/rubygem-redhat_access.spec

```diff
@@ -0,0 +1,119 @@
+%{?scl:%scl_package rubygem-%{gem_name}}
+%{!?scl:%global pkg_name %{name}}
+
+%global gem_name redhat_access
+%global plugin_name redhat_access
+
+Name: %{?scl_prefix}rubygem-%{gem_name}
+Version: 2.1.0
+Release: 1%{?foremandist}%{?dist}
+Summary: Plugin to add Redhat Access to Foreman
+Group: Applications/Systems
+License: GPLv3+
+URL: https://github.com/redhataccess/foreman-plugin
+Source0: https://rubygems.org/gems/%{gem_name}-%{version}.gem
+Requires: foreman >= 1.18.0
+Requires: %{?scl_prefix_ruby}ruby(release)
+Requires: %{?scl_prefix_ruby}ruby 
+Requires: %{?scl_prefix_ruby}ruby(rubygems) 
+Requires: %{?scl_prefix}rubygem(redhat_access_lib) >= 1.1.0
+Requires: %{?scl_prefix}rubygem(angular-rails-templates) >= 0.0.4
+Requires: %{?scl_prefix}rubygem(katello) >= 3.4.0
```

Comment:
I think this should also be a build dep

Reply:
ok, let's see if that helps instead of `foreman_tasks` (thanks for pointing out the dash vs underscore)